File: guides/micronaut-mqtt/app/src/test/java/example/micronaut/MicronautguideTest.java
package example.micronaut;

import io.micronaut.context.annotation.Property;
import io.micronaut.core.util.StringUtils;
import io.micronaut.runtime.EmbeddedApplication;
import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions;
import jakarta.inject.Inject;

@Property(name = "mqtt.enabled", value = StringUtils.FALSE)
@MicronautTest
class MicronautguideTest {

    @Inject
    EmbeddedApplication<?> application;

    @Test
    void testItWorks() {
        Assertions.assertTrue(application.isRunning());
    }

}

File: guides/micronaut-mqtt/app/kotlin/src/test/kotlin/example/micronaut/MicronautguideTest.kt
package example.micronaut

import io.micronaut.context.annotation.Property
import io.micronaut.core.util.StringUtils.FALSE
import io.micronaut.runtime.EmbeddedApplication
import io.micronaut.test.extensions.junit5.annotation.MicronautTest
import jakarta.inject.Inject
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

@Property(name = "mqtt.enabled", value = FALSE)
@MicronautTest
class MicronautguideTest {

    @Inject
    lateinit var application: EmbeddedApplication<*>

    @Test
    fun testItWorks() {
        Assertions.assertTrue(application.isRunning)
    }
}
